config/sys.config

```diff
@@ -75,6 +75,20 @@
                     {127,0,0,1}, 31341} },
    {default_routers, ["/p2p/112qB3YaH5bZkCnKA5uRH7tBtGNv2Y5B4smv1jsmvGUzgKT71QpE", "/p2p/1124CJ9yJaHq4D6ugyPCDnSBzQik61C1BqD9VMh1vsUmjwt16HNB"]},
    {mark_mods, [miner_hbbft_handler]},
-   {stabilization_period_start, 187000}
+   {stabilization_period_start, 187000},
+   {reg_domains_file, "countries_reg_domains.csv"},
+   {frequency_data, #{<<"US915">> => [903.9, 904.1, 904.3, 904.5, 904.7, 904.9, 905.1, 905.3],
+                      <<"EU868">> => [867.1, 867.3, 867.5, 867.7, 867.9, 868.1, 868.3, 868.5, 868.8],
+                      <<"EU433">> => [433.175, 433.375, 433.575],
+                      <<"CN470">> => [486.3, 486.5, 486.7, 486.9, 487.1, 487.3, 487.5, 487.7 ],
+                      <<"CN779">> => [779.5, 779.7, 779.9],
+                      <<"AU915">> => [916.8, 917.0, 917.2, 917.4, 917.5, 917.6, 917.8, 918.0, 918.2],
+                      <<"AS923">> => [923.2, 923.4, 923.6, 923.8, 924.0, 924.2, 924.4, 924.5, 924.6, 924.8],
+                      <<"KR920">> => [922.1, 922.3, 922.5, 922.7, 922.9, 923.1, 923.3],
+                      <<"IN865">> => [865.0625, 865.4025, 865.985],
+                      <<"zone1">> => [867.1, 867.3, 867.5, 867.7, 867.9, 868.1, 868.3, 868.5, 868.8],
```

Comment:
I'm confused as to the relationship between eg. US915 and zone2

Reply:
Some countries do not have a region frequency specified, in these cases we default to a zone and for each zone there is a default frequency.

Reply:
I'm not sure if we should fall back to 'zone' as I don't know how legal that is?

Reply:
If a spot ends up in a country with no specified plan how should we deal with it then ?  Its either transmit on some specified plan or block forever

Reply:
it was agreed that we should block